### PR TITLE
[Inductor Perf CI] Lower the threshold of performance smoke test speedup.

### PR DESCRIPTION
**Pull request by weiwangmeta**
Avoids issues with https://github.com/pytorch/pytorch/issues/96530

cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

File: benchmarks/dynamo/check_hf_bert_perf_csv.py [hunk 1]
```diff
@@ -18,7 +18,8 @@ def check_hf_bert_perf_csv(filename):
         speedup = row["speedup"]
         # Reduced from 1.19 to 1.17, see https://github.com/pytorch/pytorch/issues/94687
```
Comment (seemethere) [inline benchmarks/dynamo/check_hf_bert_perf_csv.py:19]: nit: should we delete earlier comments as well?

**Review by seemethere — APPROVED** [review]

File: benchmarks/dynamo/check_hf_bert_perf_csv.py (hunk 1 above)
Comment (weiwangmeta) [reply benchmarks/dynamo/check_hf_bert_perf_csv.py:19]: Good point, let me delete the earlier comments and just leave the current one there.

Comment (weiwangmeta) [reply benchmarks/dynamo/check_hf_bert_perf_csv.py]: Done!

**Pull request closed without merging**